service/orders/src/main/java/nl/tudelft/wdm/group1/orders/events/Rest.java

```diff
@@ -78,7 +78,7 @@ public void consumeOrderItemDelete(OrderItemDeletePayload payload) {
     }
 
     @KafkaHandler
-    public void checkoutOrder(OrderCheckoutPayload payload) {
```

Comment:
Is the checkout payload left unused? If it is it should be removed.

Reply:
It depends on where the bug is